docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+
+### UX and Design
+
+* [Accessibility for Designers](http://webaim.org/resources/designers/): An infographic about how designers can help create good, accessible websites.
+* [The Complete Beginner's Guide to Universal Design](http://www.uxbooth.com/articles/the-complete-beginners-guide-to-universal-design/): A blog post that describes universal design, a set of considerations made to ensure that a product, service, and/or environment is usable by everyone, to the greatest extent possible, without the need for adaptation or specialized design.
+* [Colllor](http://colllor.com/) and [0to255](http://0to255.com/): Both are tools that will generate different shades, tints and tones of colors, helpful when creating an accessible color palette.
+* [Color Palette Evaluator by NC State](http://accessibility.oit.ncsu.edu/tools/color-contrast/index.php): Evaluate the contrast of different color palettes with the Color Palette Evaluator by NC State.
+* [Tanaguru Contrast-Finder](http://contrast-finder.tanaguru.com/form.html): Find high contrast colors when you need them.
+
+### Web Developers
+
+* [aXe](https://www.deque.com/products/axe/): A browser add-on for evaluating the accessibility of web pages within the developer console.
+* [Wave](http://wave.webaim.org/): A web tool and browser add-on for evaluating the accessibility of web pages.
+* [NVDA](http://www.nvda-project.org/): A screenreader for Windows (open source).
+* [JAWS](http://www.freedomscientific.com/products/fs/jaws-product-page.asp): The most popular screenreader. Available for Windows; costs money.
+* [VoiceOver](http://www.apple.com/accessibility/voiceover/): Built-in screenreader for Mac.
+* [VoiceOver for iOS](http://www.apple.com/accessibility/iphone/vision.html): Built-in screenreader for iOS.
+* [Chrome Vox](http://www.chromevox.com/): A screenreader for ChromeOS.
+* [WCAG 2.0 Cheat Sheet](http://www.w3.org/2009/cheatsheet/#wcag2) A simplified look at WCAG 2.0.
+* [WCAG 2.0 Mind Map](http://stamfordinteractive.com.au/wp-content/uploads/2013/03/WCAG20Map.pdf): A clickable mind-map that helps put the web accessibility guidelines into perspective. This WCAG 2.0-on-a-single-page reference is an alternative way to view the guidelines.
+* [An Alt Text Decision Tree](http://dev.w3.org/html5/alt-techniques/developer.html#tree): A decision tree for deciding when and how to implement alt text in your work on the web. This is a work in progress by the editors of the HTML5 spec, but it's extremely useful in its current form.
+
+## Utilities in Calypso
+
+As we work to make Calypso more accessible, we'll probably add more things here.
+
+* [accessible-focus](client/lib/accessible-focus/README.md): A small module which is run at client startup and adds an `accessible-focus` class to the document's html element when keyboard navigation is detected, so that obvious focus styles can be added without being distracting for non-keyboard users.
```

Comment:
We also have [`Focusable`](https://github.com/Automattic/wp-calypso/tree/master/client/components/focusable) & [`ScreenReaderText`](https://github.com/Automattic/wp-calypso/tree/master/client/components/screen-reader-text) now.

Reply:
`ScreenReaderText` doesn't have a README, but I think I figured it out